easybuild/easyblocks/t/trinity.py

```diff
@@ -294,7 +294,7 @@ def install_step(self):
     def sanity_check_step(self):
         """Custom sanity check for Trinity."""
 
-        path = 'trinityrnaseq_r%s' % self.version
+        path = os.path.basename(self.cfg['start_dir'].strip('/'))
```

Comment:
this parsing of the path is copy pasted from below, where it is also used to guess PATH. so it's know to work correctly ;-)